main.go

```diff
@@ -284,8 +292,9 @@ func (tch *tileCachingHandler) ServeHTTP(w http.ResponseWriter, r *http.Request)
 		//
 		// When Trillian gets a request that is past the end of the log, it returns
 		// 400 (for better or worse), so we emulate that here.
+		tch.requestsMetric.WithLabelValues("error", "ct_log_past_end").Inc()
 		w.WriteHeader(http.StatusBadRequest)
-		w.Write([]byte("requested range is past the end of the log"))
+		fmt.Fprintln(w, "requested range is past the end of the log")
```

Comment:
Indicate the log/shard that this request was intended for.

Reply:
That will be evident to the end user based on the URL they requested. For ctile, it knows its backend by an internal name only. It will know its public-facing URL based on the request, but it's not really that useful to echo the request URL, since the end user already knows it.